MavenArtifacts.md

```diff
@@ -0,0 +1,83 @@
+# WPILib Maven Artifacts
+
+WPILib publishes its built artifacts to our maven server for use by downstream projects. This document explains these locations, and the meanings of artifact names, classifiers and versions.
+
+## Repositories
+We provide 2 repositories. These repositories are:
+
+* (Release)     http://first.wpi.edu/FRC/roborio/maven/release/
+* (Development) http://first.wpi.edu/FRC/roborio/maven/development/
+
+The release repository is where official WPILib releases are pushed.
+The development repository is where development releases of every commit to `master` is pushed.
+
+## Artifact classifiers
+We provide 2 base types of artifacts.
+
+The first types are Java artifacts. These are usually published as `jar` files. Usually, the actual jar file is published with no classifier. The sources are published with the `-sources` classifier, and the javadocs are published with the `-javadoc` classifier.
+
+The second types are Native artifacts. These are usually published as `zip` files (except for the `JNI` artifact types, which are `jar` files. See below for information on this.) The `-sources` and `-headers` classifiers contain the sources and headers respecively for the library. Each artifact also contains a classifier for each platform we publish. This platform is in the format `{os}{arch}`. The platform artifact only contains the binaries for a specific platform. In addition, we provide a `-all` classifier. This classifer combines all of the platform artifacts into a single artifact. This is useful for tools that can't determine what version to use during builds. However, we recommend using the platform specific classifier when possible. Note that the binary artifacts never contain the headers, you always need the `-headers` classifier to get those.
```

Comment:
`See below for information on this.)` -> `See below for information on this).`

I moved the period.